### PR TITLE
updated README.md

### LINKED ISSUE #10: Add documentation tailored to diffusers++
Make the documentation specifically tailored to diffusers++

### PR DESCRIPTION
**Pull request by AbhinavJangra29**
# What does this PR do?
updates readme.md

Fixes #10 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@shauray8 

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,22 +31,22 @@ limitations under the License.
...
+🔥Diffusers++🔥 is built on top of HuggingFace Diffusers, ensuring the inclusion of the latest state-of-the-art models related to image and video generation. 🔥Diffusers++🔥 is the go-to library for state-of-the-art pretrained diffusion models for generating images, audio, and even 3D structures of molecules. Whether you're looking for a simple inference solution or training your own diffusion models, 🔥Diffusers++🔥 is a modular toolbox that supports both. Our library is designed with a focus on [usability over performance](https://huggingface.co/docs/diffusers/conceptual/philosophy#usability-over-performance), [simple over easy](https://huggingface.co/docs/diffusers/conceptual/philosophy#simple-over-easy), and [customizability over abstractions](https://huggingface.co/docs/diffusers/conceptual/philosophy#tweakable-contributorfriendly-over-abstraction).
 
-🤗 Diffusers offers three core components:
+Diffusers++ offers three core components:
```
Comment (shauray8) [inline README.md:41]: Add a little reference to plus models, plus pipelines here

File: README.md [hunk 2]
```diff
@@ -31,22 +31,22 @@ limitations under the License.
...
 
 - State-of-the-art [diffusion pipelines](https://huggingface.co/docs/diffusers/api/pipelines/overview) that can be run in inference with just a few lines of code.
 - Interchangeable noise [schedulers](https://huggingface.co/docs/diffusers/api/schedulers/overview) for different diffusion speeds and output quality.
 - Pretrained [models](https://huggingface.co/docs/diffusers/api/models/overview) that can be used as building blocks, and combined with schedulers, for creating your own end-to-end diffusion systems.
 
 ## Installation
```
Comment (shauray8) [inline README.md:47]: Diffusers++ installations is a little different, for now it's through cloning only, so do add it here

Comment (shauray8) [reply README.md:47]: also we don't do flax, so no point of keeping it

File: README.md [hunk 3]
```diff
@@ -118,24 +118,32 @@ Check out the [Quickstart](https://huggingface.co/docs/diffusers/quicktour) to l
...
+- See [Duplicate](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22duplicate%22) if this issue or pull request already exists.
+- See [Enhancement](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22enhancement%22) for new feature or request.
+- See [Help wanted](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) if extra attention is needed.
+- See [Invalid](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22invalid%22) if something doesn't seem right.
+- See [Question](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22question%22) if further information is requested.
 
+Also, say 👋 in our public Discord channel <a href="https://discord.gg/HPN4TVHR"><img alt="Join us on Discord" src="https://img.shields.io/discord/823813159592001537?color=5865F2&logo=discord&logoColor=white"></a>. We discuss the hottest trends about diffusion models, help each other with contributions, personal projects or just chill out 🔥.
 
 ## Popular Tasks & Pipelines
```
Comment (shauray8) [inline README.md:148]: We can add plus pipelines here for easy access, maybe removing the others

**Review by shauray8 — CHANGES_REQUESTED** [review]
Looks good, just go through the changes I requested

File: README.md (hunk 3 above)
Comment (AbhinavJangra29) [reply README.md:148]: added

File: README.md (hunk 2 above)
Comment (AbhinavJangra29) [reply README.md:48]: done

File: README.md [hunk 4]
```diff
@@ -31,43 +31,32 @@ limitations under the License.
...
-
-### Flax
-
-With `pip` (official package):
-
-```bash
-pip install --upgrade diffusers[flax]
+git clone https://github.com/ModelsLab/diffusers_plus_plus.git
+cd diffusers_plus_plus
+pip install -e .
```
Comment (shauray8) [inline README.md:59]: Replace with `python -m pip install -e .`

**Review by shauray8 — APPROVED** [review]
Lgtm, I'll merge it with some other commits I have to push

File: README.md (hunk 4 above)
Comment (AbhinavJangra29) [reply README.md:59]: changed

**Comment by shauray8** [comment]
@adhikjoshi do check if there's anything, that should be either changed or added

**Comment by adhikjoshi** [comment]
Looks good to go

**Pull request merged**